03-sveltejs/README.md

```diff
@@ -0,0 +1,22 @@
+# Svelte
+
+In this example we have implemented the same as the previous one but using Svelte.
+
+In this case, components are written into `.svelte` files using a superset of HTML. In the `script` tag, we can write JavaScript/TypeScript code and define the `products` in a JavaScript variable using `let`.
+
+We will change the array reference when we add a new product or change the product status and using the `$` trick we will tell the compiler that the `filtered` variable will be recalculated when the `products` or `visibility` change.
```

Comment:
So when we manipulate the array and update will be triggered, and to apply changes when the filter gets updated, we need to use a _magic_ entry _$_ this is similar to the _createEffect_ in React (it will be triggered when any of the inner props used in that code changes.